ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/MonoGame.Templates.CSharp.csproj

```diff
@@ -1,26 +1,23 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <Title>MonoGame project templates</Title>
     <Description>A set of C# templates written for the MonoGame framework.</Description>
     <PackageId>MonoGame.Templates.CSharp</PackageId>
-    <PackageVersion>0.0.1</PackageVersion>
     <Authors>@MonoGameTeam</Authors>
+
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
```

Comment:
@mrhelmut You missed this one. Could you make this target only netstandard2.0?

Reply:
Done!